protocol/src/types/metadata.rs

```diff
@@ -657,21 +652,18 @@ impl Serializable for ParticleData {
 
 #[derive(Debug)]
 pub struct VillagerData {
-    villager_type: protocol::VarInt,
-    profession: protocol::VarInt,
-    level: protocol::VarInt,
+    // TODO: use these
+//villager_type: protocol::VarInt,
+//profession: protocol::VarInt,
+//level: protocol::VarInt,
 }
 
 impl Serializable for VillagerData {
-    fn read_from<R: io::Read>(buf: &mut R) -> Result<Self, protocol::Error> {
```

Comment:
Why did you remove this code?

Reply:
This code was not removed, just commented out because data from the buffer was not used.

Reply:
Okay, but are you sure this isn't causing issues because of unread bytes?